### DISCUSSION TITLE
expandable element ignores the "open" option

### DISCUSSION THREAD

**Discussion by KevinRohn** [GitbookIO/community#228] [Feature Requests]
### What's your feature/idea?

The expandable element ignores the `open` option.

**Markdown:**
```
<details open>
<summary>Expandable open on page view 😄 </summary>

Some content I want to show without collapse the expandable manually.

</details> 

<details>
<summary>Expandable open only when collapsed </summary>

Some content I want to show only when I collapse the expandable element.

</details> 
```

### What's your desired outcome?

**Preview:**
<details open>
<summary>Expandable open on page view 😄 </summary>

Some content I want to show without collapse the expandable manually.

</details> 

<details>
<summary>Expandable open only when collapsed </summary>

Some content I want to show only when I collapse the expandable element.

</details> 

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by taszap** [comment]
Related to #220

**Comment by taszap** [comment]
Closing in favour of #220
